webapp/src/containers/components/tables/existingConfigTable.tsx

```diff
@@ -0,0 +1,151 @@
+import React, {useState} from 'react';
+
+import FormGroup from 'react-bootstrap/FormGroup';
+import Col from 'react-bootstrap/Col';
+import Table from 'react-bootstrap/Table';
+import ButtonGroup from 'react-bootstrap/ButtonGroup';
+import Button from 'react-bootstrap/Button';
+
+import './styles.css';
+
+type HelpText = {
+    key: string | null;
+    props: {
+        isMarkdown: boolean;
+        isTranslated: boolean;
+        text: string;
+        textDefault?: string;
+        textValues?: string;
```

Comment:
What is the difference between `text` and `textValues`?

Reply:
There is no doccumentation provided for these props which comes in the component which is registered using registerAdminConsoleCustomSetting(). Also we are not using these fields so we are removing it